### PR TITLE
Doc how kubelet client certificate renew v5

### PR DESCRIPTION
**Pull request by jenting**
# Describe your changes

Add when/how kubelet client certificate to renew.

# Related Issues / Projects

Please provide links to Bugzilla and other GitHub projects with your description if they are related to the changes.

Relates to:

### CODE REVIEW

**Comment by jenting** [comment]
I'll create another PR for CaaSP v4~

File: adoc/admin-security-certificates.adoc [hunk 1]
```diff
@@ -534,12 +532,16 @@ if the administrator is not willing to upgrade the cluster, the `kucero` (KUbern
...
 [NOTE]
 ====
-During the kubelet client certificate signing flow, kubelet sends the kubelet client CSR into the {kube} cluster, after which the `kube-controller-manager` signs the kubelet client CSR with the Kubernetes CA cert/key pair.
+During the kubelet client/server certificate signing flow, kubelet sends the kubelet client/server CSR into the {kube} cluster. The `kube-controller-manager` signs the kubelet client CSR with the Kubernetes CA cert/key pair. The `kucero` signs the kubelet server CSR with the kubelet CA cert/key pair.
+
+The kubelet client/server certificate renews automatically at approximately 70%-90% of the total lifetime of the certificate, the kubelet would use new client/server certificates without downtime.
 
-The kubelet client deafult configuration uses a `kubelet-client-current.pem` file in the
-`/var/lib/kubelet/pki` folder. This file is a symlink to the latest generated certificate.
+The kubelet client certificate uses the `/var/lib/kubelet/pki/kubelet-client-current.pem`, this file is a symlink to the latest signed client certificate.
```
Comment (innobead) [inline adoc/admin-security-certificates.adoc:542]:
```suggestion
The kubelet client certificate is referred to `/var/lib/kubelet/pki/kubelet-client-current.pem`, this file is a symlink to the latest signed client certificate.
```

File: adoc/admin-security-certificates.adoc [hunk 2]
```diff
@@ -534,12 +532,16 @@ if the administrator is not willing to upgrade the cluster, the `kucero` (KUbern
...
+
+The kubelet server certificate uses the `/var/lib/kubelet/pki/kubelet-server-current.pem`, this file is a symlink to the latest signed server certificate.
```
Comment (innobead) [inline adoc/admin-security-certificates.adoc:544]:
```suggestion
The kubelet server certificate is referred to `/var/lib/kubelet/pki/kubelet-server-current.pem`, this file is a symlink to the latest signed server certificate.
```

File: adoc/admin-security-certificates.adoc [hunk 3]
```diff
@@ -854,7 +857,43 @@ ssh <USERNAME>@<MASTER_NODE_IP_ADDRESS/FQDN>
...
 ----
 
-=== Renewing Certificates Created by `skuba`:
+* Replace the `kubelet` server certificate:
++
+[IMPORTANT]
+====
+You need to generate `kubelet` server certificate for all the nodes on one of control plane nodes.
+The `kubelet` CA certificate key only exists on the control plane nodes.
+Therefore, after generating re-signed `kubelet` server certificate/key for worker nodes, you have to copy each `kubelet` server certificate/key from the control plane node to the corresponding worker node.
```
Comment (innobead) [inline adoc/admin-security-certificates.adoc:866]:
```suggestion
Therefore, after generating re-signed `kubelet` server certificate/key for worker nodes, you have to copy each `kubelet` server certificate/key from the control plane node to the corresponding worker nodes.
```

File: adoc/admin-security-certificates.adoc [hunk 4]
```diff
@@ -854,7 +857,43 @@ ssh <USERNAME>@<MASTER_NODE_IP_ADDRESS/FQDN>
...
+====
+. Backup the original `kubelet` certificates and keys.
++
+[source,bash]
+----
+sudo cp -r /var/lib/kubelet/pki /var/lib/kubelet/pki.bak
+----
++
+. Sign each node `kubelet` server certificate with the CA certificate/key `/var/lib/kubelet/pki/kubelet-ca.crt` and `/var/lib/kubelet/pki/kubelet-ca.key`, make sure that the signed server certificate SAN is the same as the origin.
```
Comment (innobead) [inline adoc/admin-security-certificates.adoc:875]:
```suggestion
. To sign each node `kubelet` server certificate with the CA certificate/key `/var/lib/kubelet/pki/kubelet-ca.crt` and `/var/lib/kubelet/pki/kubelet-ca.key`, make sure that the signed server certificate SAN is the same as the origin.
```

File: adoc/admin-security-certificates.adoc [hunk 5]
```diff
@@ -529,17 +527,16 @@ kubectl rollout restart deployment/oidc-gangway -n kube-system
...
 
 . *Via the `kucero` addon*:
-if the administrator is not willing to upgrade the cluster, the `kucero` (KUbernetes control plane CErtificate ROtation) addon rotates all the kubeadm managed certificates and kubeconfigs, and kubelet client and server certificates.
-`Kucero` consists of two components:
-  .. A kubeadm checker/renewer in the form of a DaemonSet, which is deployed on master nodes. It's job is to periodically check and renew control plane kubeadm managed certificates/kubeconfigs.
-  .. A kubelet server CSR signer controller, which signs the kubelet server CSR with a kubelet CA cert/key pair. The signed server certificates are stored in `/var/lib/kubelet/pki` and `/var/lib/kubelet/pki/kubelet-server-current.pem` is a symlink to the last generated server certificate.
+if the administrator is not willing to upgrade the cluster, the `kucero` (KUbernetes control plane CErtificate ROtation) addon rotates all the kubeadm managed certificates and kubeconfigs. The kucero is a kubeadm checker/renewer in the form of a DaemonSet, which is deployed on master nodes. It's job is to periodically check and renew control plane kubeadm managed certificates/kubeconfigs.
```
Comment (Klaven) [inline adoc/admin-security-certificates.adoc:530]:
```suggestion
if the administrator is not willing to upgrade the cluster, the `kucero` (KUbernetes control plane CErtificate ROtation) addon rotates all the kubeadm managed certificates and kubeconfigs. The kucero is a kubeadm checker/renewer in the form of a DaemonSet, which is deployed on control plane nodes. It's job is to periodically check and renew control plane kubeadm managed certificates/kubeconfigs.
```

File: adoc/admin-security-certificates.adoc [hunk 6]
```diff
@@ -833,8 +828,9 @@ If you are running multiple master nodes, you need to run the followings
 commands sequentially on all master nodes.
 ====
 
-=== Renewing Certificates Managed by `kubeadm`
+=== Renewing Control Plane Certificates
 
+* Replace kubeadm-managed certificates:
 . To SSH into the master node, renew all `kubeadm` certificates and reboot,
```
Comment (Klaven) [inline adoc/admin-security-certificates.adoc:834]:
```suggestion
. To SSH into the control plane node, renew all `kubeadm` certificates and reboot,
```

**Comment by jenting** [comment]
> Looks good, only gave it a quick read. a few minor updates to make our documentation more inclusive. Will read over it in more depth tomorrow!
> Thanks!

Thanks for your late reviews :bow:

**Pull request merged**